client/layout/masterbar/logged-in.jsx

```diff
@@ -660,6 +805,45 @@ class MasterbarLoggedIn extends Component {
 			return this.renderCheckout();
 		}
 
+		if ( config.isEnabled( 'layout/mb' ) ) {
+			if ( isMobile && isInEditor && loadHelpCenterIcon ) {
+				return (
+					<Masterbar>
+						<div className="masterbar__section masterbar__section--left">
+							{ this.renderMySites() }
+						</div>
+						<div className="masterbar__section masterbar__section--right">
+							{ this.renderCart() }
+							{ this.renderNotifications() }
+						</div>
+					</Masterbar>
```

Comment:
Tempted to replace this with the desktop version and just hide certain menu items when in the editor